utils/protobuf-schema-utilities/src/test/proto/TestOrderingSyntax2.proto

```diff
@@ -12,4 +12,6 @@ message Address {
 
 message Customer {
   required string name = 1;
+
+  reserved 2, 15, 10 to 11;
```

Comment:
Can you please add another test case for range that is longer? Like `10 to 11, 16 to 21`.

Reply:
Included in the extension range: #1727